client/redux/tempTypes.js

```diff
@@ -13,12 +13,13 @@ const tempTypes = [
   },
   {
     "typeId": 2,
-    "typeName": "Homeless Encampment",
-    "agencyId": 2,
-    "agencyName": "Sanitation Bureau",
-    "color": "#15BC76",
-    "description": "Encampments impacting right-of-way or maintenance of clean and sanitary public areas"
+    "typeName": "Water Waste",
+    "agencyId": 1,
+    "agencyName": "Street Lighting Bureau",
+    "color": "#F58505",
+    "description": "Water runoff, over-watering, incorrect water days, or any other water waste "
```

Comment:
Check this out https://dev-api.311-data.org/docs#/default/get_all_request_types_types_get. This is where the app is getting the list of request types from. I don't recall why this `tempTypes` file is here but we probably shouldn't be changing it or the type IDs and other info won't match what we're getting from the API.